flink-clients/src/main/java/org/apache/flink/client/program/Client.java

```diff
@@ -421,6 +426,50 @@ public JobSubmissionResult run(JobGraph jobGraph, boolean wait) throws ProgramIn
 		}
 	}
 
+	/**
+	 * Executes the CANCEL action through Client API.
+	 *
+	 * @param Accepts job id and cancels the job
+	 *
+	 */
+
+
+	public int cancel(JobID jobId) throws ProgramInvocationException {
```

Comment:
You only return 0 or 1, so you might also return a Boolean. Or just return nothing and throw an Exception in case of an error.